python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
   - name: u_bdFluxD
     "@id": "#Object.u_bdFluxD"
     datatype: double
     description: Flux from the exponential fit. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1136
   - name: u_bdFluxDErr
     "@id": "#Object.u_bdFluxDErr"
     datatype: double
     description: Flux uncertainty from the exponential fit. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1137
   - name: u_bdReB
     "@id": "#Object.u_bdReB"
     datatype: double
     description: Half-light ellipse of the de Vaucouleurs fit. Measured on u-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 1138
   - name: u_bdReD
     "@id": "#Object.u_bdReD"
     datatype: double
     description: Half-light ellipse of the exponential fit. Measured on u-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 1139
   - name: u_blendedness
     "@id": "#Object.u_blendedness"
     datatype: double
     description: Measure of how much the flux is affected by neighbors, (1 - child_flux/parent_flux).  Operates on the absolute value of the pixels to try to obtain a de-noised value.  See section 4.9.11 of Bosch et al. 2018, PASJ, 70, S5 for details. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1140
   - name: u_blendedness_flag
     "@id": "#Object.u_blendedness_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1141
   - name: u_cModelFlux
     "@id": "#Object.u_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1142
   - name: u_cModelFluxErr
     "@id": "#Object.u_cModelFluxErr"
     datatype: double
     description: Flux uncertainty from the final cmodel fit. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1143
   - name: u_cModelFlux_inner
     "@id": "#Object.u_cModelFlux_inner"
     datatype: double
     description: Flux within the fit region, with no extrapolation. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1144
   - name: u_cModel_flag
     "@id": "#Object.u_cModel_flag"
     datatype: boolean
     description: Flag set if the final cmodel fit (or any previous fit) failed. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1145
   - name: u_cModel_flag_apCorr
     "@id": "#Object.u_cModel_flag_apCorr"
     datatype: boolean
     description: Set if unable to aperture correct modelfit_CModel. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1146
   - name: u_calibFlux
     "@id": "#Object.u_calibFlux"
     datatype: double
     description: Flux within 12.0-pixel aperture. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1147
```

Comment:
Ideally we'd order this right after `u_ap12Flux`, and document it as an alias for that.